nemo/collections/nlp/data/dialogue/dataset/dialogue_bert_dataset.py

```diff
@@ -82,15 +86,150 @@ def __init__(self, dataset_split: str, dialogues_processor: object, tokenizer, c
             ignore_extra_tokens=self.cfg.ignore_extra_tokens,
             ignore_start_end=self.cfg.ignore_start_end,
         )
-
+        
         self.all_input_ids = features[0]
         self.all_segment_ids = features[1]
         self.all_input_mask = features[2]
         self.all_loss_mask = features[3]
         self.all_subtokens_mask = features[4]
         self.all_slots = features[5]
+
+
+        
+        # dict_slots = {}
+        # bio_transfer= {} #0 indicates 'O', 1 indicates 'B', 2 indicates 'I' 
+        # bio_transfer['0'] = '0'
+
+        # for idx,s in enumerate(self.all_possible_slots):
+        #     dict_slots[str(idx)]=s
+
+        # for k,v in dict_slots.items():
+        #     if 'B' in v:
+        #         bio_transfer[k] = '1'
+        #     elif 'I' in v:
+        #         bio_transfer[k] = '2'
+
+        def transfer(input_list, other_label):
```

Comment:
Have more informative names for functions - i.e. what is "transfer" seeking to do? In general, it's fine to have longer function names as long as they are informative (tell other people who read the code what the code's purpose is). Also, please do not have nested functions unless there is a good reason to (such as needing to access local variables which I don't see here). Not nesting functions allows us to test the functions. Instead you can make this method a staticmethod by having a `@staticmethod` decorator above the function name, which you would then call using `DialogueBERTDataset.transfer(...)`